skaffold.yaml

```diff
@@ -1,27 +1,33 @@
 apiVersion: skaffold/v1alpha2
 kind: Config
 
+## Default profile for use with `skaffold dev`
 build:
-
   artifacts:
   - imageName: podinfo
     docker:
       dockerfilePath: ./Dockerfile.ci
-
 deploy:
-
   kubectl: { manifests: [ deploy/skaffold/* ] }
 
 profiles:
 
-  - name: production
-
+  - name: test
+    ## This profile runs unit tests and builds the image
     build:
+      local: { skipPush: true }
+      artifacts:
+      - imageName: errordeveloper/podinfo
```

Comment:
We need to decide where this should end-up... maybe a different image in your Quay account for now or another registry?